package.json

```diff
@@ -8,7 +8,7 @@
     "start": "node app.js",
     "build": "babel . --ignore node_modules,es5-packaged,pupeteer --out-dir es5-packaged",
     "package": "npx pkg -t node16-linuxstatic-x64,node16-win-x64 -o elprimobot es5-packaged/app.js",
-    "deploy": "rsync  -avz elprimobot-linuxstatic elprimobot:~/development/",
+    "deploy": "rsync  -avz elprimobot-linuxstatic elprimobot:~/elprimobot/",
```

Comment:
Was this change intended to be here? Feels unrelated.

Reply:
Thanks I've noticed the deployment path was wrong, and sneakily fixed it :D will add to the PR description